packages/application-shell/src/components/login-sso/__snapshots__/login-sso.spec.js.snap

```diff
@@ -39,7 +39,7 @@ exports[`rendering <Formik> form elements should match layout 1`] = `
         onClick={[MockFunction]}
         size="big"
         tone="primary"
-        type="button"
+        type="submit"
```

Comment:
snapshots 🤔

Reply:
He's just gonna say that on every PR like a bot.